Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Dockerfile for setting up reviewdog and ktlint
+
+FROM ubuntu:latest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag](https://app.codacy.com/manual/davidmerrick/AdventOfCode2019/pullRequest?prid=4961687)